dftimewolf/lib/state.py

```diff
@@ -399,8 +400,10 @@ def _RunModuleProcessThreaded(
         as executor:
       pop = not module.KeepThreadedContainersInState()
       for c in self.GetContainers(module.GetThreadOnContainerType(), pop):
-        logger.debug(f'Launching {module.name}.Process thread with {str(c)} '
-            f'from {c.src_module_name}')
+        logger.debug(
+            f'Launching {module.name}.Process thread with {str(c)} from '
```

Comment:
Similarly, I'd opt to update c to maybe container or something a bit more obvious

Reply:
Same as above, line the fstring placeholder on 405 goes too long with a longer variable name.